src/pages/data-globe/data-globe-selectors.js

```diff
@@ -17,6 +18,14 @@ const getUiSettings = createSelector(selectUiUrlState, uiUrlState => {
   }
 })
 
+const getSceneSettings = createSelector(getGlobeSettings, globeSettings => {
+  return {
+    ...sceneSettings,
+    zoom: globeSettings.zoom,
```

Comment:
Shouldn't this params belong to `globeSettings`? As they are params that change the way the `globe` appears?

Reply:
As we agreed, I'll take care of handling state changes in the next PR.